kaldb/src/main/java/com/slack/kaldb/writer/kafka/KaldbKafkaWriter.java

```diff
@@ -102,19 +102,20 @@ void consumeMessages(long kafkaPollTimeoutMs) throws IOException {
     KafkaConsumer<String, byte[]> consumer = getConsumer();
 
     ConsumerRecords<String, byte[]> records = consumer.poll(Duration.ofMillis(kafkaPollTimeoutMs));
-    LOG.debug("Fetched records." + records.count());
-
     int recordCount = records.count();
-    recordsReceivedCounter.increment(recordCount);
-    int recordFailures = 0;
-    for (ConsumerRecord<String, byte[]> record : records) {
-      if (!messageWriter.insertRecord(record)) recordFailures++;
+    LOG.debug("Fetched records={}", recordCount);
```

Comment:
nothing changed. Just made the `OG.debug("Processed {} r` statement print only when there were messages received.

Helps in debugging test failures